ClosedXML_Tests/Excel/Tables/TablesTests.cs

```diff
@@ -1114,6 +1114,21 @@ public void TableNotFound()
             }
         }
 
+        [Test]
+        public void SecondTableOnNewSheetHasUniqueName()
+        {
+            using (var wb = new XLWorkbook())
+            {
+                var ws1 = wb.AddWorksheet();
+                var t1 = ws1.FirstCell().InsertTable(Enumerable.Range(1, 10).Select(i => new { Number = i }));
+                Assert.AreEqual("Table1", t1.Name);
+
+                var ws2 = wb.AddWorksheet();
+                var t2 = ws2.FirstCell().InsertTable(Enumerable.Range(1, 10).Select(i => new { Number = i }));
+                Assert.AreEqual("Table2", t2.Name);
+            }
+        }
+
```

Comment:
It seems, the reference file `AddingDataSet.xlsx` has to be updated: it contained `Table1`, `Table11`, and `Table12` (not 1, 2, 3). I wonder how did we miss that...

Reply:
I've noticed that before. Just never had the motivation to chase the problem.